### ISSUE TITLE
How to repeat notification

### ISSUE THREAD

**Issue by romsworld** [nielsfaber/alarmo#516]
Hi,

I would like to repeat notification for laps time, every 5s for exemple on my phone
Alarm... Alarm... Alarm

Only 1 is not suffisant.
I try this, but it's not working. 

service: notify.mobile_app_myphone
service_data:
  message: TTS
  title: 'Alarm '
  data:
    ttl: 0
    priority: high
    channel: alarm_stream_max
    delay:
      hours: 0
      minutes: 0
      seconds: 5
      milliseconds: 0

Thanks,

**Comment by nielsfaber** [comment]
You can create an automation in HA that repeats an action while the alarm is triggered.
This is not something you can setup within alarmo.
Here is an example: https://community.home-assistant.io/t/automation-loop-until-state-changes/388382/3

**Comment by andjones132** [comment]
I have this same request. Your suggestion to use a standard ha automation script works ok if I just want to send a generic message, but it gets more complicated if I also want to include the wildcard of the friendly name of the sensor that triggered the Alarm. I am sure there is probably some template function that I could use to pull the name of the sensor from the Attribute of the Alarm when in Triggered State. However, this adds extra overhead when the built in capability of Alarmo already has almost 90% of what I need. Alarmo already has the ability through Actions to send messages via notify to mobile app or text-to-speech with the wildcard name of the sensor. However, to the best of my knowledge, it currently does not have the ability to repeat the message x number times with an x number of second delay between each message. If it is 2 am and the Alarm goes off, and Alarmo sends a TTS announcement to all the media players in the house, I doubt I will hear the whole message the first time. It would be nice if there were two more options:

1.	Number of times to repeat the message: e.g. 3
2.	Delay in seconds between messages: e.g. 15

My current workaround is to turn on the MQTT option in Alarmo and have Node-RED listen for a Triggered event. It can then use the information in the MQTT event to pull out the friendly name of the sensor that triggered the alarm and use the Loop function to repeat the message x number of times before triggering the siren. The work around works but with risks that it could break at any time by some “breaking change”. One of the reasons I switched to Alarmo is that must have missed some breaking change from several months ago broke my homemade NR notifications. I only discovered it when I finally got around to doing a full systems test and then figuring out how far back I need to go to find a version of HA prior to the breaking change. It would be nice if this was built-in to help reduce the number of breakpoints that any Node-RED or HA automation would add. Thank you for your consideration.

On a personal note, I wanted to Thank You for the tremendous amount of time and effort you have put into Alarmo. It must be at the top of the list for the most helpful and potentially lifesaving integrations that available for home assistants. With my old Node-RED alarm automation, I had to worry every time I hit the upgrade button in HA if a known or unknown “breaking change” would break a system I rely on for the health and well-being of my family. I understand the "Use at your own risk" clause, but I am not sure why anyone would spend the time, energy, and money to set up HA and create a custom home security solution if they didn’t intend to rely on it. It would be fantastic if the HA Dev team would recognize your contribution and make Alarmo a fully supported HA addon. Then, we could have some confidence that it would be a part of the test package run prior to each release to ensure it s working as intended. I will happily be contributing to your Patron/Paypal account regularly, if for no other reason than for the time it saves me having to maintain my own solution. Thanks again, and keep up the great work.
